packaging/docker/custom/Dockerfile

```diff
@@ -29,7 +29,7 @@ ENV GF_PLUGIN_RENDERING_CHROME_BIN="/usr/bin/chromium-browser"
 RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
     grafana-cli \
         --pluginsDir "$GF_PATHS_PLUGINS" \
-        --pluginUrl https://github.com/grafana/grafana-image-renderer/releases/latest/download/plugin-linux-x64-glibc-no-chromium.zip \
+        --pluginUrl https://github.com/grafana/grafana-image-renderer/releases/latest/download/plugin-linux-x64-musl-no-chromium.zip \

```

Comment:
we should probably detect whether the base is using musl or glibc and set this appropriately, we can either use the same alpine detection we now use in the main dockerfile or try to detect whether glibc is available directly